### PR TITLE
First try!

### PR DESCRIPTION
**Pull request by kakurokan**

### CODE REVIEW

**Comment by kakurokan** [comment]
Merging is blocked???

File: PhoneBook.Kakurokan/Program.cs [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+
+            return new(name, email, number);
+        }
+
+        public static void DisplayReturningTomenu()
+        {
+            AnsiConsole.Clear();
+            AnsiConsole.Status().Start("Returning to menu...", ctx =>
+            {
+                Thread.Sleep(2000);
```
Comment (antonderegt) [inline PhoneBook.Kakurokan/Program.cs:179]: Haha is this an exercise in Zen or mindfulness 🤣 . Nice touch with the loading screen, but it took for ever 😆

**Review by antonderegt — APPROVED** [review]
Hey @kakurokan! Awesome, you implemented all the requirements! I like the way your console looks, it's very clean.

Btw, you use .NET 7, it will be out of support next month 😉 time to update 😆 .

You did a good job. If you want and extra challenge. Try to look up separation of concers and implement it in your next projects.

APPROVED!

**Comment by antonderegt** [comment]
> Merging is blocked???

Yea that's expected, don't worry about it, it's only possible after review. 😉

**Comment by kakurokan** [comment]
Thank you for the review!! Good to know about .NET7, I will update as soon as possible. About the separation of concerns, I will look up and implement in my next projects.

**Pull request closed without merging**